vagrant/provisioning/roles/httpd/templates/httpd-alfresco-redirect.conf

```diff
@@ -277,24 +277,24 @@ SSLCertificateChainFile "{{ ssl_ca }}"
 
 
 # Reverse proxy settings for Arkcase and components
-ProxyPass /alfresco https://{{ internal_host }}:7070/alfresco
-ProxyPassReverse /alfresco https://{{ internal_host }}:7070/alfresco
+#ProxyPass /alfresco https://{{ internal_host }}:7070/alfresco
+#ProxyPassReverse /alfresco https://{{ internal_host }}:7070/alfresco
 
-<LocationMatch "/alfresco">
-  <If '%{REMOTE_ADDR} != {{ arkcase_host_address | default('127.0.0.1') }}'>
-    RequestHeader unset X-Alfresco-Remote-User
-  </If>
-</LocationMatch>
+#<LocationMatch "/alfresco">
+#  <If '%{REMOTE_ADDR} != {{ arkcase_host_address | default('127.0.0.1') }}'>
```

Comment:
Do we need this file any more?  If everything is commented out let's just delete the whole file.